### ISSUE TITLE
Feature: cannot add alt text to images

### ISSUE THREAD

**Issue by nf3xn** [trunks-social/trunks-public#293]
For accessibility we need to be able to add alt text to images.

**Steps to reproduce:**
None feature missing

**What we expect to happen:**
Some mechanism for adding the text, while the image or video is being uploaded

**Comment by Decad3nce** [comment]
You have to wait until the upload completes, then you can add the alt text - it is supported.
